lib/timers.js

```diff
@@ -229,7 +234,7 @@ function tryOnTimeout(timer, list) {
   timer._called = true;
   var threw = true;
   try {
-    timer._onTimeout();
+    ontimeout(timer);
```

Comment:
Try deopt any container function. Node.js should, IMHO, always test for nullity and use flags/optionals and never try-catch in the inner functions... This pattern is not really a requirement in FP and there's huge performance benefits on native calls.

Not really a Node commiter, but I have been experimenting this kind of crack.